recipes/msdf-atlas-gen/all/test_package/conanfile.py

```diff
@@ -0,0 +1,26 @@
+from conans import ConanFile, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    @ property
+    def _atlas_texture_file(self):
+        return os.path.join(self.source_folder, "atlas_texture.png")
```

Comment:
Always create artifacts in the build folder.
Otherwise multiple runs of the test package will interfere with each other.
```suggestion
        return os.path.join(self.build_folder, "atlas_texture.png")
```

Reply:
Oh, that's probably why tests failed so (I didn't check the full log yet).